src/newsletter-editor/typography/index.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * WordPress dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { compose } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+import { withDispatch, withSelect } from '@wordpress/data';
+import { Fragment, useEffect } from '@wordpress/element';
+import { SelectControl } from '@wordpress/components';
+
+const fontOptions = [
+	{
+		value: null,
+		label: __( 'Sans Serif', 'newspack-newsletters' ),
+		disabled: true,
+	},
+	{
+		value: 'Arial',
+		label: __( 'Arial', 'newspack-newsletters' ),
+	},
+	{
+		value: 'Tahoma',
+		label: __( 'Tahoma', 'newspack-newsletters' ),
+	},
+	{
+		value: 'TrebuchetMS',
+		label: __( 'Trebuchet', 'newspack-newsletters' ),
+	},
+	{
+		value: 'Verdana',
+		label: __( 'Verdana', 'newspack-newsletters' ),
+	},
+	{
+		value: null,
+		label: __( 'Serif', 'newspack-newsletters' ),
+		disabled: true,
+	},
+	{
+		value: 'Georgia',
+		label: __( 'Georgia', 'newspack-newsletters' ),
+	},
+	{
+		value: 'Palatino',
+		label: __( 'Palatino', 'newspack-newsletters' ),
+	},
+	{
+		value: 'TimesNewRoman',
+		label: __( 'Times New Roman', 'newspack-newsletters' ),
+	},
+	{
+		value: null,
+		label: __( 'Monospace', 'newspack-newsletters' ),
+		disabled: true,
```

Comment:
Why are these three disabled option in the select? Also, two of them are not in the backend list of fonts.

Reply:
`Serif`, `Sans Serif` and `Monotype` are category groupings, not actual fonts. It's meant to match [this screenshot](https://user-images.githubusercontent.com/177929/81283646-e2815200-9054-11ea-8c96-c64c06ff305f.png), although without the indentation. Actually all three of these should be absent from the list of fonts—which was the one you did find?

Reply:
[`'Serif'` is on the PHP list](https://github.com/Automattic/newspack-newsletters/pull/210/files#diff-76aefac9a481217d439a23e638846effR31)

Reply:
> category groupings

`SelectControl` does not support [`optgroup`](https://www.w3schools.com/TAGs/tag_optgroup.asp) :/

Reply:
> `'Serif'` is on the PHP list

Whoops: https://github.com/Automattic/newspack-newsletters/pull/210/commits/7bb8e8b866aba8c631ff609729fb71859d49b350